packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -2497,3 +2516,31 @@ const findBookingQuery = async (bookingId: number) => {
   // Don't leak any sensitive data
   return foundBooking;
 };
+
+function isLoggingEnabled({
```

Comment:
Logging can be enabled either using `BOOKING_LOGGING_EVENT_IDS` by specifying comma separated list of event ids or by `BOOKING_LOGGING_USER_OR_TEAM_NAME` by specifying comma separated list of username or teamname